vllm/utils.py

```diff
@@ -230,8 +230,9 @@ def hpu_backend_string():
 
 @lru_cache(maxsize=None)
 def _is_habana_frameworks_installed() -> bool:
-    from importlib import util
-    return util.find_spec('habana_frameworks') is not None
+    if hasattr(torch, "hpu"):
```

Comment:
Since this is check if habana_frameworks is installed/loaded, we should simplify to return has hasattr(torch, "hpu")
Just note, the change relies on the fact that when habana_frameworks is loaded, the hpu module will appear in torch